lib/osf-components/addon/components/osf-link/styles.scss

```diff
@@ -0,0 +1,5 @@
+.OsfLink {
+    &:hover {
+        color: var(--secondary-color);
```

Comment:
Based on our meeting, should this be primary color?

Reply:
yeah i think that's what we said